### ISSUE TITLE
NativeKeyboard & Ionic Keyboard Conflict

### ISSUE THREAD

**Issue by jayordway** [EddyVerbruggen/cordova-plugin-native-keyboard#98]
I have points in my app where I used both Eddy's native keyboard plugin as well as the conventional keyboard provided with ionic. In the situation that I go to a spot in the app that uses the native keyboard plugin, when I go to another point in the app that used only the conventional ionic keyboard, the native keyboard pops up instead. If I do not use the native keyboard first, the correct keyboard appears when I click on inputs and expect the ionic keyboard.

Basically, after I use this native keyboard plugin in one spot in my app, it is the keyboard that pops up everywhere - even in areas that i would prefer to fallback to the regular ionic keyboard. How can this be prevented?
